src/custom/state/claim/hooks/utils.ts

```diff
@@ -151,3 +154,25 @@ function _repoNetworkIdMapping(id: SupportedChainId): string {
 export function getClaimKey(account: string, chainId: number): string {
   return `${chainId}:${account}`
 }
+
+export type PaidClaimTypeToPriceMap = {
+  [type in ClaimType]: { token: Token; amount: string } | undefined
+}
+
+/**
+ * Helper function to get vCow price based on claim type and chainId
+ *
+ * @param type
+ */
+export function claimTypeToTokenAmount(type: ClaimType, chainId: SupportedChainId) {
+  switch (type) {
+    case ClaimType.GnoOption:
+      return { token: GNO[chainId], amount: GNO_PRICE }
+    case ClaimType.Investor:
+      return { token: USDC, amount: USDC_PRICE }
```

Comment:
Note to self: yes, USDC is only meant for mainnet, but we do want to test it on rinkeby, right?

Reply:
100%, we should use it in the 3 networks